_layouts/person.html

```diff
@@ -4,6 +4,9 @@
 ---
 {% assign person = page %}
 {% capture submit_link %}/collect/person/?name={{person.path_name}}{% endcapture %}
+{% if person.graduated %}
+  {% capture year_link %}/years/{{ person.graduated | minus: 1 | slice: 2,2 }}_{{ person.graduated | slice: 2,2 }}/{% endcapture %}
```

Comment:
Can we put this in `person.rb`? Hesitant to use liquid for stuff like this as it tends to break in weird ways when we upgrade.

Reply:
Done.